syntax="proto2";

package proto;

message PartP {
  required int32 partKey = 1;
  required string name = 2;
  required string mfgr = 3;
  required string brand = 4;
  required string type = 5;
  required int32 size = 6;
  required string container = 7;
  required double retailPrice = 8;
  required string comment = 9;
}
  

  
  



